src/Faker/Provider/cs_CZ/Person.php

```diff
@@ -425,6 +425,117 @@ class Person extends \Faker\Provider\Person
         'Bc.', 'Ing.', 'MUDr.', 'MVDr.', 'Mgr.', 'JUDr.', 'PhDr.', 'RNDr.', 'doc.', 'Dr.'
     );
 
+    protected static $birthNumbers = array(
```

Comment:
Why these numbers and not others? Why doesn't it rely on a `numerify()` pattern?